src/components/ItemEditorPage/RightPanel/MultiSelect/MultiSelect.tsx

```diff
@@ -50,7 +50,7 @@ export default class MultiSelect<T extends string> extends React.PureComponent<P
         <div className="values">
           {value.length > 0 ? (
             value.map(value => (
-              <div className="value">
+              <div className="value" key={value}>
```

Comment:
Maybe we should be using the `index` here? because values on a multiselect aren't necessarily unique right?

Reply:
they are actually :) if you try to write the same tag twice you will notice the second one doesn't get added (like if it was a Set), that's why i used value as a unique id